src/components/signup/NameForm.js

```diff
@@ -28,7 +28,7 @@ class NameForm extends React.Component<Props, State> {
   state = {
     errorMessage: '',
     fullName: this.props.screenProps.data.fullName || '',
-    isValid: true,
+    isValid: false,
```

Comment:
I think will cause the empty text to show as invalid no?

Reply:
No, button is active